docs/_docs/framework/capabilities/manage/practice-ops.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: default
+grand_parent: FinOps Framework
+parent: Manage
+title: FinOps Practice Operations
+permalink: /framework/capabilities/manage/practice-ops
+nav_order: zzz
```

Comment:
Should we follow the order of the FinOps Foundation? 

For the Manage the FinOps Practice domain the have the capabilities in the following order: 

1. FinOps Practice Operations
2. Cloud Policy & Governance
3. FinOps Assessment 
4. FinOps Tools & Services
5. FinOps Education & Enablement
6. Invoicing & Chargeback 
7. Onboarding Workloads
8. Intersecting Disciplines